.github/workflows/test.yml

```diff
@@ -171,8 +171,8 @@ jobs:
       run: PYTHONPATH=. DEBUG=2 python3 extra/torch_backend/torch_tests.py TestTinyBackendPRIVATEUSE1.test_unary_log_tiny_float32
     - name: Test Ops with TINY_BACKEND (expect failure)
       run: PYTHONPATH=. LLVM=1 LLVMOPT=0 TINY_BACKEND=1 python3 -m pytest -n auto test/test_ops.py || true
-    - name: Test beautiful_mnist in torch with TINY_BACKEND (expect failure)
-      run: PYTHONPATH=. TINY_BACKEND=1 python3 examples/other_mnist/beautiful_mnist_torch.py || true
+    - name: Test beautiful_mnist in torch with TINY_BACKEND
+      run: PYTHONPATH=. ALLOW_DTYPE_MISMATCH=1 TINY_BACKEND=1 python3 examples/other_mnist/beautiful_mnist_torch.py
```

Comment:
`ALLOW_DTYPE_MISMATCH` no longer needed. Also, can you add `TARGET_EVAL_ACC_PCT=96.0` to the test and check that? Then I think the bounty is yours!